### ISSUE TITLE
Column Background Types

### ISSUE THREAD

**Issue by zackpyle** [beaverbuilder/feature-requests#180]
I would like to be able to have the same background options for Columns that we do for Rows (Color, Gradient, Photo, Video, Embedded Code, etc)
![Screen Shot 2021-08-10 at 13 58 11](https://user-images.githubusercontent.com/19413506/128919288-f8389555-3c6d-4933-bc3a-9690dda8b79e.jpg)

Currently only Color, Gradient, and Photo are available for Columns ->
![Screen Shot 2021-08-10 at 13 59 02](https://user-images.githubusercontent.com/19413506/128919310-90c214e0-a85e-4927-87e7-6bd8a74dd2f7.jpg)

**Comment by scamando** [comment]
Yes, please add this. The workaround I have been using is: Save a video background and contents saved row. Then load that row in your column using HTML module + saved row shortcode.

**Comment by zackpyle** [comment]
@Pross At least adding the Embedded Code option to columns would enable us to add the others via code if desired... 🙏🏻

**Comment by zackpyle** [comment]
@Pross Any ideas here?

**Comment by zackpyle** [comment]
@codente No longer under consideration?
